app/modules/import-export/assets/js/admin.js

```diff
@@ -1,19 +1,231 @@
-const revertButton = document.getElementById( 'elementor-import-export__revert_kit' );
+class Admin {
+	/**
+	 * Session Storage Key
+	 *
+	 * @type {string}
+	 */
+	KIT_DATA_KEY = 'elementor-kit-data';
 
-if ( revertButton ) {
-	revertButton.addEventListener( 'click', ( event ) => {
+	/**
+	 * Contains the ID of the referrer Kit and the name of the Kit to remove. Stored in session storage.
+	 *
+	 * @type {Object}
+	 */
+	cachedKitData;
+
+	/**
+	 * The 'Remove Kit' revert button
+	 *
+	 * @type {Element}
+	 */
+	revertBtn;
+
+	/**
+	 * Name of the kit currently active (last imported)
+	 *
+	 * @type {string}
+	 */
+	activeKitName;
+
+	constructor() {
+		this.activeKitName = this.getActiveKitName();
+		this.revertBtn = document.getElementById( 'elementor-import-export__revert_kit' );
+
+		if ( this.revertBtn ) {
+			this.revertBtn.addEventListener( 'click', this.revertBtnOnClick.bind( this ) );
+			this.maybeAddRevertBtnMargin();
+		}
+
+		this.maybeShowReferrerKitDialog();
+	}
+
+	/**
+	 * Add bottom margin to revert btn if referred from Kit library
+	 */
+	maybeAddRevertBtnMargin() {
+		const referrerKitId = new URLSearchParams( this.revertBtn.href ).get( 'referrer_kit' );
+		if ( ! referrerKitId ) {
+			return;
+		}
+
+		this.revertBtn.style.marginBottom = this.calculateMargin();
+		this.scrollToBottom();
+	}
+
+	/**
+	 * CalculateMargin
+	 *
+	 * @return {string}
+	 */
+	calculateMargin() {
+		const adminBar = document.getElementById( 'wpadminbar' );
+		const adminBarHeight = adminBar ? adminBar.offsetHeight : 0;
+
+		const revertKitHeight = this.revertBtn.parentElement.offsetHeight;
+
+		return (
+			document.body.clientHeight -
+			adminBarHeight -
+			revertKitHeight -
+			document.getElementById( 'wpfooter' ).offsetHeight -
+			15 // Extra margin at the top
+		) + 'px';
+	}
```

Comment:
OK, after understanding the feature better:

1. I can't merge it like this, we need to tell the product that this UI isn't gonna happen. I talked with @nivnoimanelementor about it.
2. It doesn't work like the Figma anyway, you scroll the whole body instead of the inner wrapper.
3. The best I can suggest is adding some highlight effect to the button somehow (add class on page load?), you can suggest it to the product.

Reply:
This is Ok by me. I approve it 👍